### ISSUE TITLE
Add hard-coded support for UTCTime

### ISSUE THREAD

**Issue by bitc** [cdepillabout/pretty-simple#91]
This library is great, but doesn't work when it encounters a [UTCTime](https://hackage.haskell.org/package/time-1.11.1.1/docs/Data-Time-Clock.html#t:UTCTime):

```haskell
    now <- liftIO getCurrentTime
    pPrint (now, now)
```

This is the result:

```
( 2021 - 03 - 16 18 : 04 : 08.725928 UTC
, 2021 - 03 - 16 18 : 04 : 08.725928 UTC
)
```

For reference, this is what `Show` for UTCTime looks like

```
(2021-03-16 18:04:08.725928 UTC,2021-03-16 18:04:08.725928 UTC)
```

I think it would be a good idea to hardcode support for UTCTime into this library. Some type of regex could detect the string fragments that look like UTCTime, and keep them the way they are (and possibly color them in a distinct color)

Thank you

**Comment by georgefst** [comment]
See #75.

TL;DR - we used to handle it better, and we can hopefully get back there without hardcoding anything.

**Comment by cdepillabout** [comment]
Lets close this in favor of #75.
